### PR TITLE
fix 0.1.1 warnings

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by fenollp** [comment]
Helps with https://github.com/elli-lib/elli_websocket/pull/5

File: src/elli_ws_request_adapter.erl [hunk 1]
```diff
@@ -66,6 +66,13 @@
 
 -type req() :: #req_adapter{}.
 
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
```
Comment (yurrriq) [inline src/elli_ws_request_adapter.erl]: Should we use something like this, as in ellli?

```erlang
{erl_opts, [{platform_define, "^2[1-9]", post20}]}.
```

Comment (fenollp) [reply src/elli_ws_request_adapter.erl]: Hm if you want me to I can do it but I believe it amounts to the same.

Comment (yurrriq) [reply src/elli_ws_request_adapter.erl]: I guess, I'd prefer to be explicit about it.

**Comment by fenollp** [comment]
amended

**Comment by yurrriq** [comment]
LGTM thanks! With this are we ready for the 0.1.1 release?

**Pull request merged**